go/libraries/doltcore/sqle/tables.go

```diff
@@ -432,6 +439,15 @@ func (t *DoltTable) GetForeignKeys(ctx *sql.Context) ([]sql.ForeignKeyConstraint
 	return toReturn, nil
 }
 
+/*func (t *DoltTable) WithProjection(colNames []string) sql.Table {
+	t.projectedCols = colNames
+	return t
+}
+
+func (t *DoltTable) Projection() []string {
+	return t.projectedCols
+}*/
+
```

Comment:
Disabled until I have a chance to look into an issue with colNames being off in some cases.